apps/frontend/src/components/SettingsPage.tsx

```diff
@@ -183,6 +192,49 @@ function SettingsPage() {
         <DialogTitle>{dialogTitle}</DialogTitle>
         <DialogContent>
           <DialogContentText>{dialogMsg}</DialogContentText>
+          {isDialogButtonEnabled ? (
+            <Box>
+              {isDialogInputEnabled ? (
```

Comment:
A cleaner way to write this is `isDialogInputEnabled && <Box...`
https://dev.to/harlessmark/short-circuit-evaluation-with-react-3dn4